Fix layouts

Ant-Nazif opened the pull request:
* Fix layout padding
* Added a new layout for Claim manager

Ant-Nazif committed:
CHECKPOINT/Fix bugs and added a new layout for Claim manager

kkuna commented on src/selectors/identity.js:
```diff
@@ -26,7 +26,7 @@ export const selectClaims = (state) => state.identity.getIn(['personalInformatio
   [selectClaimCategories, selectShowEmptyClaimCategories, selectClaims],
   (claimCategories, showEmptyClaimCategories, claims) => {
-    if (!showEmptyClaimCategories) {
+    if (showEmptyClaimCategories) {
```
This should not be changed, we can only invert the logic for checked prop on the checkbox.

kkuna committed:
UPDATE / Show empty categories by default

--------------------------------------------------------------------
 MAIN
 * Show empty claim categories by default
--------------------------------------------------------------------
